ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/anomalylocalization/HybridCounter.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ *
+ */
+
+package org.opensearch.ml.engine.algorithms.anomalylocalization;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.extern.log4j.Log4j2;
+
+/**
+ * A hybrid counter that starts with exact counting with map and switches to approximate counting with sketch as the size grows.
+ */
+@Log4j2
+public class HybridCounter implements Counter {
+
+    protected static int SKETCH_THRESHOLD = 10_000;
+
+    private Counter counter = new Hashmap();
```

Comment:
How about renaming class `Hashmap` as `HashMapCounter` or `ExactCounter` ? `Hashmap` looks confusing as java has `HashMap`

Reply:
+1

Reply:
i renamed it to HashMapCounter